backend/core/src/auth/constants.ts

```diff
@@ -1,3 +1,26 @@
+import { CookieOptions } from "express"
 // Length of hashed key, in bytes
 export const SCRYPT_KEYLEN = 64
 export const SALT_SIZE = SCRYPT_KEYLEN
+export const TOKEN_COOKIE_NAME = "access-token"
+export const REFRESH_COOKIE_NAME = "refresh-token"
+export const ACCESS_TOKEN_AVAILABLE_NAME = "access-token-available"
+export const TOKEN_COOKIE_MAXAGE = 86400000 // 24 hours
```

Comment:
Maybe not for this story, but this could be a good candidate to be an environment variable in case future sites want a longer/shorter login period

Reply:
I agree!